showcase/src/assets/examples/table/with-items-per-page-selector/table.html

```diff
@@ -0,0 +1,12 @@
+<stark-table
+	[data]="data"
+	[columnProperties]="columns"
+	[filter]="filter"
+	[orderProperties]="order"
+	[paginationConfig]="pagination"
+	[tableRowActions]="tableRowActions"
+	multiSort
+	showRowsCounter
+>
+	<header><h1 class="mb0" translate>SHOWCASE.DEMO.TABLE.WITH_ITEMS_PER_PAGE_SELECTOR</h1></header>
```

Comment:
Could you please replace the translation key by "Table with the selector of items per page to display" ?
As we have in the other examples 😊

Reply:
Yes, I was also wondering, isn't there any way to use the translation in the example too ?

Reply:
No, there is no way because the `example-viewer` component displays the raw file.

Reply:
ok, I see, crap ...